### ISSUE TITLE
Get a list of recipes within python

### ISSUE THREAD

**Issue by nsheff** [databio/refgenconf#83]
The CLI has a way to show the list of available recipes.

I cannot find out how to do that from the python API.

**Comment by stolarczyk** [comment]
you're right there's no way to get that from the python API. It's not trivial since `asset_build_packages` are a part of refgenie package.

This will require some thought

**Comment by nsheff** [comment]
maybe this is an indication that those build packages should move into refgenconf.

**Comment by nsheff** [comment]
I believe we actually thought about this before, but decided not to change it due to plans to revamp the recipe system anyway.

but I believe the integration with galaxy will require the python API to be able to do this. We will definitely need to do this at some point

**Comment by nsheff** [comment]
Related to #75

**Comment by nsheff** [comment]
implemented via RecipeManager in refgenie1.
